src/ripe_rainbow/interactive/drivers.py

```diff
@@ -227,16 +227,9 @@ def click(self, element, ensure = True):
         return element
 
     def ensure_visible(self, element, timeout = None):
-        # waits until the element is visible from a document point of view
-        # no scroll ensuring a z index collision is enforced
-        self._wait(timeout = timeout).until(
-            lambda d: element.is_displayed(),
-            "Element never became visible at block (display and opacity)"
-        )
-
         # starts the operation by moving the cursor to the outside of the element
         # this ensures that the cursor is not "moving over" the element
-        self._move_outside(element)
+        self._move_outside(element, raise_e = False)

```

Comment:
Firefox always raises exceptions when trying to move to elements outside the viewport.